packages/orchestration/src/exos/stakingAccountKit.js

```diff
@@ -217,7 +350,60 @@ export const prepareStakingAccountKit = (baggage, makeRecorderKit, zcf) => {
           if (!result) throw Fail`Failed to delegate.`;
           return tryDecodeResponse(result, MsgDelegateResponse.fromProtoMsg);
         },
+        /**
+         * Only `bondDenom` deposits accepted until #9211, #9063
+         * @param {Payment} payment
+         * @param {{ timeoutTimestamp: bigint }} [opts]
+         * @returns {Promise<{ sequence: number }>}
+         */
+        async deposit(payment, opts) {
+          const brand = await E(payment).getAllegedBrand();
+          const issuer = this.state.brandToIssuer.get(brand);
+          issuer || Fail`Unknown Issuer for Brand ${brand}.`;
+
+          const amount = await E(issuer).getAmountOf(payment);
+          !AmountMath.isEmpty(amount) ||
+            Fail`Payment amount must be greater than 0.`;
 
+          const { localAccount } = this.state;
+          trace('Depositing funds to LCA');
+          // XXX consider adding exposing an interface to withdraw / send
+          // messages from the LCA (e.g., withdraw funds)
+          await E(localAccount).deposit(payment);
+
+          const timeoutTimestamp =
```

Comment:
this is a conditional await. I think it's safe because of the await above it, but please add a `// @jessie-check` at the top to be sure. all modules that run in a vat should pass jessie-check